qutip/core/cy/qobjevo.pyx

```diff
@@ -323,11 +345,29 @@ cdef class QobjEvo:
         """Return a copy of this `QobjEvo`"""
         return QobjEvo(self, compress=False)
 
-    def arguments(QobjEvo self, dict new_args):
-        """Update the arguments"""
+    def arguments(QobjEvo self, dict _args=None, **kwargs):
+        """
+        Update the arguments.
+
+        parameter
+        ---------
+        args : dict [positional / optional]
+            New arguments as a dict. Update args with ``arguments(new_args)``.
+
+        **kwargs :
+            New arguments as a keywors. Update args with
+            ``arguments(**new_args)``.
+
+        .. note:
+            If both the positional ``args`` and keywords are passed new values
+            from both will be used. If a key is present with both, the ``args``
+            dict value will take priority.
+        """
+        if _args:
+            kwargs.update(_args)
```

Comment:
```suggestion
        if _args is not None:
            kwargs.update(_args)
```
If by any chance `_args=0` this will not raise an error and it should. Small thing that will most likely neve occur but best to have it.